revolv/templates/administrator/email.html

```diff
@@ -1,42 +1,20 @@
 {% extends "base/base.html" %}
 {% load staticfiles %}
 
+{% block head %}
+    <link rel="stylesheet" type="text/css" href="{% static "email.css" %}" />
+    <link href='https://fonts.googleapis.com/css?family=Lato:300' rel='stylesheet' type='text/css'>
+{% endblock %}
+
 {% block title %}Home{% endblock %}
```

Comment:
@sameerav I know this line wasn't changed, but would you mind updating it to have the actual correct title? haha